### ISSUE TITLE
Joining server causes client to freeze and stop responding V1.2.1

### ISSUE THREAD

**Issue by DjordjeMandic** [xJon/Tekkit-2#206]
**Basic details**
- **Are you using a legitimate Minecraft account?**: yes (same behaivour on cracked clients)
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**: [RAR Compressed to 32KB because log file is 45MB](https://file.io/E0Nq26iNPNs9)

**Describe the issue**

Sometimes joining server causes client to freeze, server registeres connection and player joins but client is frozen on "Logging in" screen and turns grey(not responding). After connection has timed out this is what shows up in console repeated a lot in single second(cant scroll up to find normal logs). Server still works. Sometimes im unable to join even after restarts but restoring backed up world fixes it until this happens again pretty soon. 

```
[21:01:34] [Netty Server IO #16/ERROR] [FML]: NetworkDispatcher exception
java.io.IOException: An existing connection was forcibly closed by the remote host
        at sun.nio.ch.SocketDispatcher.writev0(Native Method) ~[?:1.8.0_371]
        at sun.nio.ch.SocketDispatcher.writev(Unknown Source) ~[?:1.8.0_371]
        at sun.nio.ch.IOUtil.write(Unknown Source) ~[?:1.8.0_371]
        at sun.nio.ch.SocketChannelImpl.write(Unknown Source) ~[?:1.8.0_371]
        at io.netty.channel.socket.nio.NioSocketChannel.doWrite(NioSocketChannel.java:432) ~[minecraft_server.1.12.2.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:856) [minecraft_server.1.12.2.jar:?]
        at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.forceFlush(AbstractNioChannel.java:368) [minecraft_server.1.12.2.jar:?]
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:618) [minecraft_server.1.12.2.jar:?]
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:559) [minecraft_server.1.12.2.jar:?]
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:476) [minecraft_server.1.12.2.jar:?]
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438) [minecraft_server.1.12.2.jar:?]
        at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [minecraft_server.1.12.2.jar:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_371]
```

**Comment by DjordjeMandic** [comment]
This is client log moment before it freezes

```
[21:15:14] [Client thread/INFO] [craftingtableiv]: Initialised 6094 recipes in 215 ms
[21:15:14] [Client thread/INFO] [FML]: Applying holder lookups
[21:15:14] [Client thread/INFO] [FML]: Holder lookups applied
[21:15:14] [Netty Client IO #1/INFO] [FML]: [Netty Client IO #1] Client side modded connection established
[21:15:14] [Netty Client IO #1/INFO] [HammerCore]: Registering raytrace plugin: tk.zeitheron.cableflux.intr.hammercore.IntegrHammerCore ...
[21:15:14] [Netty Client IO #1/INFO] [HammerCore]: Registered raytrace  plugin: tk.zeitheron.cableflux.intr.hammercore.IntegrHammerCore in 3 ms
[21:15:14] [Netty Client IO #1/INFO] [HammerCore]: Registering raytrace plugin: com.zeitheron.hammercore.internal.blocks.multipart.HammerCoreRaytracePlugin ...
[21:15:14] [Netty Client IO #1/INFO] [HammerCore]: Registered raytrace  plugin: com.zeitheron.hammercore.internal.blocks.multipart.HammerCoreRaytracePlugin in 0 ms
```

**Comment by DjordjeMandic** [comment]
Played for few days without resource pack, its client related issue with resource pack.

**Comment by xJon** [comment]
This is the same issue as https://github.com/xJon/The-1.12.2-Pack/issues/876, and it will be resolved in the next update.
See https://github.com/xJon/The-1.12.2-Pack/issues/876#issuecomment-1512895802 for a workaround (so you can keep using a resource pack).

**Comment by xJon** [comment]
Should be fixed in `1.2.2`.
